hconfig_utils/tests/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+set(MODULE_DIRECTORY "${esma_include}/hconfig_utils/tests")
+
+set (test_srcs
+  Test_hconfig_get_private.pf
+  )
+
+
+add_pfunit_ctest(MAPL.hconfig_utils.tests
+                TEST_SOURCES ${test_srcs}
+                LINK_LIBRARIES MAPL.hconfig_utils MAPL.shared MAPL.pfunit
+                EXTRA_USE MAPL_pFUnit_Initialize
+                WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+                MAX_PES 4
+                )
+set_target_properties(MAPL.hconfig_utils.tests PROPERTIES Fortran_MODULE_DIRECTORY ${MODULE_DIRECTORY})
+
+if (APPLE)
+  set(LD_PATH "DYLD_LIBRARY_PATH")
+else()
+  set(LD_PATH "LD_LIBRARY_PATH")
+endif ()
```

Comment:
Note to @mathomp4  Would it make sense to set this globally for GEOS.   (Possibly with longer name.)